src/app/devices/kvm/kvm.component.html

```diff
@@ -1,3 +1,16 @@
-<app-device-toolbar [deviceState]="deviceState" ></app-device-toolbar>
+<mat-toolbar>
+    <div fxFlex="25">
+        <mat-form-field appearance="fill" class="encoding-options">
```

Comment:
remove `appearance="fill"` and `class="encoding-options"`.  We aren't using this style anywhere on the site and want to stay consistent.

Reply:
Typicall we don't want to have a toolbar inside a toolbar. Place this toolbar after the `app-device-toolbar` with only the drop down, or perhaps ideally the user is prompted via a dialog box upon kvm connecting allowing them to choose.

Reply:
done